server/webdriver_tests/browser_test.py

```diff
@@ -186,8 +188,107 @@ def test_page_serve_ca_population(self):
+        typeOf_subtitle = self.driver.find_element_by_xpath(
+            '//*[@id="node"]/h2[2]')
+        self.assertEqual(typeOf_subtitle.text, 'typeOf: BiologicalSpecimen')
+
+        # Assert properties contains correct dcid and typeOf
+        element_present = EC.presence_of_element_located(
+            (By.XPATH, '//*[@id="node-content"]/div[1]/div/table'))
+        WebDriverWait(self.driver, self.TIMEOUT_SEC).until(element_present)
+        table = self.driver.find_element_by_xpath(
+            '//*[@id="node-content"]/div[1]/div/table')
+        dcid_row = table.find_elements_by_xpath('.//tbody/tr[2]/td')
+        self.assertEqual(dcid_row[0].text, 'dcid')
+        self.assertEqual(dcid_row[1].text, 'dc/bsmvthtq89217')
+        typeOf_row = table.find_elements_by_xpath('.//tbody/tr[3]/td')
+        self.assertEqual(typeOf_row[0].text, 'typeOf')
+        self.assertEqual(typeOf_row[1].text, 'BiologicalSpecimen')
+        self.assertEqual(typeOf_row[2].text, 'nybg.org')
+
+        # Assert image loaded.
+        element_present = EC.presence_of_element_located(
+            (By.ID, 'browser-image-section'))
+        WebDriverWait(self.driver, 2 * self.TIMEOUT_SEC).until(element_present)
+        image_section = self.driver.find_element_by_id('browser-image-section')
+        image = image_section.find_element_by_tag_name('img')
+        self.assertTrue(image)
+
+    def test_stat_var_hierarchy(self):
+        """Test that the stat var hierarchy can search properly"""
+        # Load MTV browser page.
+        self.driver.get(self.url_ + MTV_URL)
+
+        # Wait for the search box of the statvar hierarchy section to be present
+        element_present = EC.presence_of_element_located(
+            (By.XPATH,
+             '//*[@id="stat-var-hierarchy-section"]/div/div[1]/input'))
+        WebDriverWait(self.driver, self.TIMEOUT_SEC).until(element_present)
+        search_input = self.driver.find_element_by_xpath(
+            '//*[@id="stat-var-hierarchy-section"]/div/div[1]/input')
+
+        # Search for "male asian " and select the first result
+        search_input.send_keys(SEARCH_INPUT)
+        element_present = EC.presence_of_element_located((
+            By.XPATH,
+            '//*[@id="stat-var-hierarchy-section"]/div/div[1]/div[2]/div/div[1]'
+        ))
+        WebDriverWait(self.driver, self.TIMEOUT_SEC).until(element_present)
+        first_result = self.driver.find_element_by_xpath(
+            '//*[@id="stat-var-hierarchy-section"]/div/div[1]/div[2]/div/div[1]'
+        )
+        first_result.click()
+
+        # Assert that the section Count_Person_Male_AsianAlone opened and shows at least one chart
+        element_present = EC.presence_of_element_located((
+            By.XPATH,
+            '//*[@id="stat-var-hierarchy-section"]/div/div[2]/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div/div[11]/div/div/div/div/div/div[1]/div[2]'
```

Comment:
this is incredibly brittle... are there classes you could add to make this easier to update in the future (e.g. is-open)?

Reply:
done

Reply:
i wonder if it could be simplified more, but this is much better